### PR TITLE
Docs: Modify the description of autoRotateSpeed.

### PR DESCRIPTION
**Pull request by 1993heqiang**
**Description**
Modify the description of autoRotateSpeed.

### CODE REVIEW

File: docs/examples/en/controls/OrbitControls.html [hunk 1]
```diff
@@ -87,7 +87,7 @@ <h3>[property:Boolean autoRotate]</h3>
 		<h3>[property:Float autoRotateSpeed]</h3>
 		<p>
 			How fast to rotate around the target if [page:.autoRotate] is true. Default is 2.0, which equates to 30 seconds
-			per rotation at 60fps.<br> Note that if [page:.autoRotate] is enabled, you must call [page:.update]
+			per round at 60fps.<br> Note that if [page:.autoRotate] is enabled, you must call [page:.update]
```
Comment (Mugen87) [inline docs/examples/en/controls/OrbitControls.html:90]: English is not my first language so I'm not sure if `round` is an improvement over `rotation`. Wouldn't it be more clear to use the term `per orbit`?

Comment (1993heqiang) [reply docs/examples/en/controls/OrbitControls.html:90]: image

This word is the word used in the description of `autoRotateSpeed` ​​in OrbitControls.js. My first language is Chinese. However, I think `round `is better understood than `orbit`.  😄

Comment (Mugen87) [reply docs/examples/en/controls/OrbitControls.html:90]: I believe `orbit` is the technical correct term. But if the inline docs already established `round`, I'm fine with it.

**Review by Mugen87 — APPROVED** [review]

File: docs/examples/en/controls/OrbitControls.html (hunk 1 above)
Comment (WestLangley) [reply docs/examples/en/controls/OrbitControls.html:90]: Yes, `orbit` is best in US English -- or `rotation`. `round` is not common.

Comment (1993heqiang) [reply docs/examples/en/controls/OrbitControls.html:90]: @mrdoob What's your opinion？

Comment (mrdoob) [reply docs/examples/en/controls/OrbitControls.html:90]: I think @WestLangley's suggestion sounds good.

Comment (1993heqiang) [reply docs/examples/en/controls/OrbitControls.html:90]: OK. I will submit it again later.

Comment (1993heqiang) [reply docs/examples/en/controls/OrbitControls.html]: Done, thank you for your suggestions. @Mugen87 @WestLangley @mrdoob

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**